docs/source/core_concepts/glossary.rst

```diff
@@ -259,15 +258,15 @@ Transaction Status Set
 ^^^^^^^^^^^^^^^^^^^^^^
 
  - NOT_RECEIVED: requested peer does not have this transaction.
+ - ENOUGH_SIGNATURES_COLLECTED: this transaction is a multisignature transaction which has enough signatures and is going to be validated by the peer.
```

Comment:
this ~transaction~ is a multisignature transaction

Reply:
This status is returned to non-mst transactions also (even with mst_enabled=false)